base/system/diskpart/diskpart_msg.mc

```diff
@@ -576,23 +577,24 @@ Example:
     CLEAN
 .
 Language=Romanian
-    Removes any and all partition or volume formatting from the disk with
-    focus.
+    Șterge orice și întreaga formatare a partiției sau a volumului din
+    discul în cauză.
 
-Syntax:  CLEAN [ALL]
+Sintaxă:  CLEAN [ALL]
 
-    ALL         Specifies that each and every byte\sector on the disk is set to
-                zero, which completely deletes all data contained on the disk.
+    ALL         Specifică faptul că fiecare octet/sector de pe disc este setat
+                pe zero, ceea ce șterge complet toate datele conținute pe disc.
 
-    On master boot record (MBR) disks, only the MBR partitioning information
-    and hidden sector information are overwritten. On GUID partition table
-    (GPT) disks, the GPT partitioning information, including the Protective
-    MBR, is overwritten. If the ALL parameter is not used, the first 1MB
-    and the last 1MB of the disk are zeroed. This erases any disk formatting 
-    that had been previously applied to the disk. The disk's state after 
-    cleaning the disk is 'UNINITIALIZED'.
+    Pe discurile cu Registrul principal de inițializare (MBR), sunt suprascrise
+    numai informațiile despre partiționarea MBR și informațiile despre
+    sectoarele ascunse. Pe discurile cu tabel de partiție GUID (GPT),
+    informațiile de partiționare GPT, incluzând MBR protectiv, sunt
+    suprascrise. Dacă parametrul ALL nu e folosit, primul 1 Mo și ultimul 1 Mo
+    al discului sunt puse la valoarea zero. Aceasta șterge orice formatare a
+    discului ce a fost aplicată anterior discului. Starea discului după ce
+    este curățat discul este 'NEINIȚIALIZAT'.
```

Comment:
This is just a comment calling for clarification. The string "UNINITIALIZED", describing disk state, at the moment of writing this, is appearing as such only in this text, nowhere else (in the entire ReactOS), and that is odd considering that in this text it is just a reference. I presume it should make its appearance at least somewhere else (like on the inspection of the partition state)? Will the rest of partition states be available for translation? (Depending on the answer to the last question, this text should or shouldn't be left untranslated.)

Reply:
I'll leave the word untranslated between quotes and I will translate it also in brakets.